oc-chef-pedant/pedant_config.rb

```diff
@@ -204,3 +204,6 @@
 
 # Log HTTP Requests
 log_file "/var/log/opscode/oc-chef-pedant/http-traffic.log"
+
+include_version_in_status false
```

Comment:
Do these tests pass when we set these two values to `true`?

Reply:
It fails because the default value is false. If we set the variables in the chef-server.rb to true, and set here also true, then it will pass.

The default values in default.rb
`default['private_chef']['opscode-erchef']['include_version_in_status'] = false`
`default['private_chef']['opscode-erchef']['include_x_ops_api_info'] = false`